config/jobs-chromeos.yaml

```diff
@@ -435,6 +435,15 @@ jobs:
       branch:
         - for-kernelci
 
+  kselftest-cpufreq:
+    template: kselftest.jinja2
+    kind: test
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
+      collections: cpufreq
+      job_timeout: 10
+    kcidb_test_suite: kselftest.cpufreq
+
```

Comment:
This would be better off in `pipeline.yaml` as it definitely isn't chrome{os,book}-specific (and we already have `kselftest-dt` defined there).

LGTM otherwise, thanks!

Reply:
Thanks @a-wai, I've forced pushed the changes.